test-runner/src/main/scala/org/typesafe/spark/mesos/framework/runners/MesosIntegrationTestRunner.scala

```diff
@@ -17,9 +17,8 @@ object MesosIntegrationTestRunner {
     printMsg("TestResults:")
     result.foreach(printMsg)
 
-    //TODO: Generate some JUnit style report for CI
-    if (result.exists(_.startsWith("TestFailed"))) {
-      System.exit(1)
+    if (result.exists(_.contains("FAILED"))) {
+      printMsg(Console.RED + "ERROR: One or more tests failed" + Console.RESET)
```

Comment:
Without I get:

```
ERROR: One or more tests failed

[success] Total time: 859 s, completed Nov 24, 2015 2:48:11 PM
```

with the `exit`:

```
ERROR: One or more tests failed

Exception: sbt.TrapExitSecurityException thrown from the UncaughtExceptionHandler in thread "run-main-0"
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
[trace] Stack trace suppressed: run last compile:runMain for the full output.
[error] (compile:runMain) Nonzero exit code: 1
[error] Total time: 258 s, completed Nov 24, 2015 3:23:14 PM
```

For me, the second one is more clear that something didn't work.
(it might be also because I am colorblind, and at a glance, I cannot make the difference between the red and the green line of text ...)

Maybe we could have something a bit cleaner, but for me, it is a good first step.